section8_function_inputs/project/main.py

```diff
@@ -0,0 +1,66 @@
+alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+
+def encrypt(message, shifting, alphabet_length = len(alphabet)):
+    caesar_coding = ""
+    # Looping every letter in text.
+    for each_letter in range(len(message)): 
+        encrypt_letter = message[each_letter]
+        # If the letter is in alphabet list, encrypt program is going to encode each letter.
+        if message[each_letter] in alphabet:
+            message_index_num = alphabet.index(message[each_letter])
+            # Using for loop to figure out the actual index number of encoded letter in list.
+            for sequence in range(shifting):
+                message_index_num+=1
+                # If the index is greater than final index number on the right hand side then return message_index_num into 0.
+                if message_index_num == alphabet_length:
+                    message_index_num = 0
+            encrypt_letter = alphabet[message_index_num]
```

Comment:
overall, I think you need to simplify this function.

suggestions
- remove unnecessary variables
- figure out the way to minimize for loop
- rename the variables to be more precise
- 
hints
- it can be done without range() function at all